### PR TITLE
add docker

### PR DESCRIPTION
**Pull request by simonostendorf**

### CODE REVIEW

**Review by TitusKirch — CHANGES_REQUESTED** [review]
For the `docker-compose.yml` i haven't mark all change request. Have a look at this repo https://github.com/laravel/sail/tree/1.x/stubs and use the variables frrom the env with the same syntax i comment for the db setttings

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      OCTANE_HTTPS: true # set to false if you want to test without https
+      OCTANE_WORKERS: 4
+      OCTANE_MAX_REQUESTS: 512
+      LOG_CHANNEL: stack
+      LOG_LEVEL: debug
+      DB_CONNECTION: mysql
+      DB_HOST: mariadb
+      DB_PORT: 3306
+      DB_DATABASE: laravel
+      DB_USERNAME: laravel
```
[the remark this answers is not in the document]
Comment (simonostendorf) [reply docker-compose.yml]: fixed

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      APP_URL: https://127.0.0.1:8000/
+      OCTANE_HTTPS: true # set to false if you want to test without https
+      OCTANE_WORKERS: 4
+      OCTANE_MAX_REQUESTS: 512
+      LOG_CHANNEL: stack
+      LOG_LEVEL: debug
+      DB_CONNECTION: mysql
+      DB_HOST: mariadb
+      DB_PORT: 3306
+      DB_DATABASE: laravel
```
[the remark this answers is not in the document]
Comment (simonostendorf) [reply docker-compose.yml]: fixed

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      DB_USERNAME: laravel
+      DB_PASSWORD: USE_SECRET_STRING_HERE
```
[the remark this answers is not in the document]
Comment (simonostendorf) [reply docker-compose.yml]: fixed

File: .github/workflows/docker-release.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+name: Docker CI (release)
+
+on:
+  push:
```
[the remark this answers is not in the document]
Comment (simonostendorf) [reply .github/workflows/docker-release.yml]: fixed

File: .github/workflows/docker-release.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      - uses: actions/checkout@v3
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Log into registry ${{ env.REGISTRY }}
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - run: |
```
[the remark this answers is not in the document]
Comment (simonostendorf) [reply .github/workflows/docker-release.yml]: fixed

File: .github/workflows/docker-dev.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: actions/checkout@v3
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Log into registry ${{ env.REGISTRY }}
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - run: |
```
[the remark this answers is not in the document]
Comment (simonostendorf) [reply .github/workflows/docker-dev.yml]: fixed

**Review by TitusKirch — APPROVED** [review]

**Pull request merged**